src/hal/getpinmap_heltec_lora32.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+
+Module:  getpinmap_heltec_lora32.cpp
+
+Function:
+        Arduino-LMIC C++ HAL pinmap for Heltec WiFi LoRa 32 (V1 & V2) and Heltec Wireless Stick
+
+Copyright & License:
+        See accompanying LICENSE file.
+
+Author:
+        Manuel Bleichenbacher, manuel.bleichenbacher@gmail.com   October 2019
+
+*/
+
+#include <arduino_lmic_hal_boards.h>
+#include <Arduino.h>
+
+#include "../lmic/oslmic.h"
+
+// Note: The pin constants SS, RST_LoRa and DIOx are defined in pins_arduino.h
+// (board-specific variant in Arduino core). Even if it won't be used, this
+// file needs to compile for all other variants as well.
+#if !defined(ARDUINO_HELTEC_WIFI_LORA_32) && !defined(ARDUINO_HELTEC_WIFI_LORA_32_V2) && !defined(ARDUINO_HELTEC_WIRELESS_STICK)
```

Comment:
I ran into an issue when trying to use this code on my Heltec WiFi LoRA ESP32 (v2), and it turned out that the Arduino IDE was defining `ARDUINO_WIFI_LORA_32_V2`, not `ARDUINO_HELTEC_WIFI_LORA_32_V2` (no HELTEC).  I assume that's because the boards.txt file here: https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/blob/master/esp32/boards.txt#L186 specifies `wifi_lora_32_V2.build.board=WIFI_LORA_32_V2`.  
The other thing that's close the Heltec package seems to define for that board is here: https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/blob/master/esp32/variants/wifi_lora_32_V2/pins_arduino.h#L6 (`#define WIFI_LoRa_32_V2 true`).
 I've never really got deep into Arudino, but am I missing something here?  Should I be setting something somewhere in the IDE, or defining `ARDUINO_HELTEC_WIFI_LORA_32_V2` in my own sketch or build arguments somewhere?
Thanks!

Reply:
It comes from `boards.txt`. Since this worked at one time, my guess is that the `boards.txt` file changed. We can add the checks for the new constant. (You can patch yourself by changing the file here to add `ARDUINO_WIFI_LoRa_32_V2` here, and also in the .h file:
https://github.com/mcci-catena/arduino-lmic/blob/540034cd2bc3373f2c05a17b2a9e13a0de1056d4/src/hal/getpinmap_thisboard.cpp#L60

Reply:
Thanks for the quick reply.  I got my board working by substituting `s/ARDUINO_HELTEC_WIFI_/ARDUINO_WIFI_/`in the two files in arduino-lmic/src/hal/..., but I wondered what the "right" solution was, since it seemed to work for everyone but me (I didn't see any posts anywhere with the same issue I was having), but computers are often like that for me :-).